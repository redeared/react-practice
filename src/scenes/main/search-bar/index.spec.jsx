import { create } from 'react-test-renderer';
import React from 'react';
import { SearchBar } from './index';

describe('SearchBar', () => {
    it('should render without crash', () => {
        const comp = create(<SearchBar filters={{ title: false, genre: false }} />).toJSON();
        expect(comp).toMatchSnapshot();
    });
});
