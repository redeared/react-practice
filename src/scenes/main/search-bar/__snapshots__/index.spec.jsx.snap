// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar should render without crash 1`] = `
<div
  style={
    Object {
      "background": "rgba(0, 0, 0, 0.7)",
      "color": "white",
      "paddingLeft": "4em",
      "paddingRight": "4em",
    }
  }
>
  <div>
    FIND YOUR MOVIE
  </div>
  <div>
    <input
      onChange={[Function]}
      onKeyPress={[Function]}
      type="text"
    />
  </div>
  <div>
    SEARCH BY: 
    <button
      onClick={[Function]}
      style={null}
    >
      TITLE
    </button>
    <button
      onClick={[Function]}
      style={null}
    >
      GENRE
    </button>
    <div
      style={
        Object {
          "float": "right",
        }
      }
    >
      <button
        onClick={[Function]}
      >
        SEARCH
      </button>
    </div>
  </div>
</div>
`;
